server/src/metadata/object-metadata/object-metadata.service.ts

```diff
@@ -57,6 +58,55 @@ export class ObjectMetadataService extends TypeOrmQueryService<ObjectMetadataEnt
       isActive: true,
       isCustom: true,
       isSystem: false,
+      fields:
+        // Creating default fields.
+        // No need to create a custom migration for this though as the default columns are already
+        // created with default values which is not supported yet by tenant migrations.
+        [
+          {
+            type: FieldMetadataType.UUID,
```

Comment:
@Weiko For default fields we can maybe imagine an util that can be used here and in seeds to avoid re-declaring the whole object each time.
This util should get optional parameter to pass, the ids and the objectMetadataId when needed